libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.resolver.ts

```diff
@@ -54,10 +46,10 @@ export class NationalRegistryXRoadResolver {
     @Context('req') { user }: { user: User },
     @Parent() person: NationalRegistryPerson,
   ): Promise<NationalRegistryPerson[] | undefined> {
-    return this.nationalRegistryXRoadService.getChildrenCustodyInformation(
-      user,
-      person.nationalId,
-    )
+    if (person.nationalId !== user.nationalId) {
```

Comment:
Wondering if we should handle this in GraphQL fields using the filterFields Extension. Check identity.model in api/domains/identity for reference.

Since this can apply to more fields resolvers than this one.

Reply:
As it is, users can't query anyone except their-self from this domain. This code is not concerned with filtering fields, as I don't know which fields should be filtered should this domain be opened up more.

This code is just dealing with the fact that - with/without field filtering - it is impossible to query children for another person.

Reply:
I think it makes sense to use the filterFields extension if/when we open this domain up. It would be part of that project to decide which fields are available.